dev/env/kafka.env

```diff
@@ -35,7 +35,7 @@ KAFKA_NUM_NETWORK_THREADS=3
 KAFKA_NUM_IO_THREADS=3
 
 # Configure default log cleanup. You can override these on per-topic basis
-KAFKA_LOG_CLEANUP_POLICY=compact
+# KAFKA_LOG_CLEANUP_POLICY=compact
```

Comment:
Setting the `KAFKA_LOG_CLEANUP_POLICY` to `compact` prevents to send messages without a key, according to this post (https://stackoverflow.com/questions/51114355/corrupt-message-when-trying-to-run-a-kafka-jdbc-source-connector).